bridges/YoutubeBridge.php

```diff
@@ -315,111 +448,6 @@ private function parseJSONListing($jsonData)
         }
     }
 
-    public function collectData()
-    {
-        $xml = '';
-        $html = '';
-        $url_feed = '';
-        $url_listing = '';
```

Comment:
Could you please place collectDataInternal to this place, in order to save "git blame" thing?